addons/binding/org.openhab.binding.lgwebos/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="lgwebos" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="WebOSTV">
+		<label>WebOS TV</label>
+		<description>WebOS based smart TV</description>
+
+		<channels>
+			<channel id="power" typeId="commandType">
```

Comment:
yes, TV powers off when items changes from ON to OFF. When the TV turns on the binding will detect it a few seconds later. It will then update this channel and set it to ON.

Reply:
My question is more about when the switch is in OFF state and I push it in the UI. It will send ON command and show it is in ON state until a state update is received from the binding. If ON command will actually not turn the TV on, the real TV state will be OFF, but the UI will show it as ON. I don't know if the state will be updated from the TV. If not, it may need some code in the power control to handle this.

Reply:
whenever the tv turns on or off the channel will be updated correctly.

however, if somebody switches to ON in the UI the switch will go to ON, but nothing happens on the tv. This Is not supported LG TVs (when they are off, there is no api). On some models, like mine it can be solved with the Wake On LAN binding as suggested in the README documentation.

Reply:
Would it work if for the ON command, we simply update the channel state to the current TV state (which should be OFF really, but someone may start the TV from the outside in the meantime)? This should cause the switch to return to OFF.

Reply:
I added autoupdate="false" to the demo item. that should do the trick.